AEPServices/Sources/network/NetworkServiceConstants.swift

```diff
@@ -26,4 +26,20 @@ public struct NetworkServiceConstants {
     public struct HeaderValues {
         public static let CONTENT_TYPE_URL_ENCODED = "application/x-www-form-urlencoded"
     }
+    
+    public static let RECOVERABLE_NETWORK_TRANSPORT_ERROR_CODES = [
+        NetworkTransportErrors.NSURLErrorTimedOut,
```

Comment:
Can you reuse the system provided enums instead of defining new constants? 
[UrlError.timedOut.rawValue](https://developer.apple.com/documentation/foundation/urlerror/2293002-timedout)

Reply:
@praveek 
I saw there are NSURLError enums as well [NSURLErrorTimedOut](https://developer.apple.com/documentation/foundation/1508628-url_loading_system_error_codes/nsurlerrortimedout). These are direct Ints so no need for any conversion.